src/models/formFields.ts

```diff
@@ -0,0 +1,17 @@
+export interface FormFields {
+  username: string;
+  name: string;
+  email: string;
+  age: number;
+  skills: any;
+  commitment: number;
+  additionalInfo: string;
+  resumeUrl: string;
+  experience: string;
+  reason: string;
+  accepted: boolean;
+  zip: number;
+  role: string;
+  id: string;
```

Comment:
I don't see this in the newApplicant schema

Reply:
Im not sure what this comment is referring to.

Reply:
id